.github/CODEOWNERS

```diff
@@ -1940,6 +1940,10 @@ x-pack/plugins/observability_solution/synthetics/server/saved_objects/synthetics
 /.github/workflows/oblt-github-commands   @elastic/observablt-robots @elastic/kibana-operations
 /.github/workflows/undeploy-my-kibana.yml @elastic/observablt-robots @elastic/kibana-operations
 
+# CODEOWNERS reviews are not required on backports
+# Assign kibanamachine to avoid notifications
+* @kibanamachine
```

Comment:
Isn't that an entry worth for the `ULTIMATE_PRIORITY_RULES`  in the generation to guarantee that it's actually always last in this file?

Reply:
Pushed be3d9f2de4b457d05572b5b1583ee8cb5d67774a